docs/api/hospital.md

```diff
@@ -57,30 +62,80 @@ print(result.json())
-            "name": "Consultorio Daniel Hale Williams",
-            "patron": "Consultorio Daniel Hale Williams",
-            "location": "Paseo de la Reforma , Cuauhtémoc, 06500 Ciudad de México",
+            "name": "Consultorio de la Dra. Matilde Montoya",
+            "patron": "Consultorio de la Dra. Matilde Montoya",
+            "location": "Nombre de la Calle, Colonia, CP, Alcaldia, Estado",
             "contact": "hola@prescrypto.com",
-            "headline": "LLegando a tu corazon desde 1893",
+            "headline": "",
             "page_size": false,
-            "picture": "https://www.prescrypto.com/media/Cardiologia.jpg",
-            "university_logo": "https://www.prescrypto.com/media/UNAM.jpg",
+            "page_template": "classic",
+            "picture": "https://www.prescrypto.com/media/logo.jpg",
+            "university_logo": "https://www.prescrypto.com/media/university.jpg",
             "is_active": true,
             "is_admin": true,
             "info": true,
-            "location_lat": "19.42755149734502",
-            "location_lon": "-99.16844323991225",
-            "zip_code": "06500"
+            "send_emails": true,
+            "zip_code": "06500",
+            "location_lat": "00.0000000000000",
+            "location_lon": "00.0000000000000",
+            "google_place_id": ""
         }
     ]    
 }
 ```
 
-### List a Single Hospital
+```json title="[Error: 401] Invalid Token"
+{
+    "message": "Invalid Token"
+}
+```
+
 You can also filter the information for a single Hospital, by adding the Hospital id to the endpoint route.
 
+#### GET Single Hospital
 ```python title="GET /api/v2/hospital/{{Hospital_ID}}"
-{{Base_URL}}/api/v2/hospital/1/
+import requests
+import json
+# Assign the necessary parameters for the request (url and parameters payload)
+url = "{{Base_URL}}/api/v2/hospital/{{Hospital_ID}}"
+
+headers = {
+  'Authorization': 'Token {{YOUR_TOKEN}}'
+}
+
+result = requests.get(url=url, headers=headers)
+print(result.json())
+```
+
+#### Response to Single Hospital
+```json title="[StatusCode: 200] Success here are the details of the Hospital of choice"
+{
+    "id": 1,
+    "created_at": "2016-01-01T04:22:12Z",
+    "name": "Consultorio de la Dra. Matilde Montoya",
+    "patron": "Consultorio de la Dra. Matilde Montoya",
+    "location": "Nombre de la Calle, Colonia, CP, Alcaldia, Estado",
+    "contact": "hola@prescrypto.com",
+    "headline": "",
+    "page_size": false,
+    "page_template": "classic",
+    "picture": "https://www.prescrypto.com/media/logo.jpg",
+    "university_logo": "https://www.prescrypto.com/media/university.jpg",
+    "is_active": true,
+    "is_admin": true,
+    "info": true,
+    "send_emails": true,
+    "zip_code": "06500",
+    "location_lat": "00.0000000000000",
+    "location_lon": "00.0000000000000",
+    "google_place_id": ""
+}
+```
+
+```json title="[Error: 404] Hospital not found or your token do not created the Hospital"
```

Comment:
Este mensaje puede confundir a mi parecer

Algo como: 
Hospital not found or you don't have access to it.

Reply:
Listo el cambio @xtornasol512